nicegui/elements/input.py

```diff
@@ -52,6 +54,25 @@ def toggle_type(_):
 
         self.validation = validation
 
+        if autocomplete is not None:
+            def AutoCompleteInput():
+                if len(self.value) > 1:
```

Comment:
Why do you only start autocompletion after the second character typed in?

Reply:
Ok, without `if` is a bit too much. But I think `> 0` would be more intuitive. Take vscode autocompletion for example. It starts suggesting as soon as you start typing.